app/components/RelatedRecordTable.tsx

```diff
@@ -27,7 +27,7 @@ export default function RelatedRecordTable<T extends Record<string, unknown>>({
     <Card>
       <Card.Header>{title}</Card.Header>
       <Card.Body>
-        <Table striped hover>
+        <Table striped hover responsive>
```

Comment:
This fixes the table overflowing its card on smaller displays 🙏 

<img width="751" alt="Screenshot 2025-01-13 at 1 40 29 PM" src="https://github.com/user-attachments/assets/96154739-43ef-49bf-8093-2731361d5b9e" />